test/acceptance/user_announcement_test.exs

```diff
@@ -54,53 +54,56 @@ defmodule ConstableWeb.UserAnnouncementTest do
 
     session
     |> visit(announcement_path(Endpoint, :edit, announcement, as: user.id))
-    |> fill_in("announcement_title", with: "Updated title")
-    |> fill_in("announcement_body", with: "# Updated")
+    |> fill_in(announcement_title(), with: "Updated title")
+    |> fill_in(announcement_body(), with: "# Updated")
     |> click_submit_button
 
     assert has_announcement_title?(session, "Updated title")
     assert has_announcement_interest?(session, "elixir")
   end
 
+  defp announcement_title  do
```

Comment:
Can these be constants too, like in `test/acceptance/user_manages_interests_test.exs`?

Reply:
Yeah, I thought about making them all like module attributes, but I wasn't sure if it made more sense like that or just as functions. I'll update these.

Reply:
Updated these in 847c0eb5d274a83e9a1075ec1b0af7a5c7d4753f